### PR TITLE
Support colors for buttons and links

### PR DESCRIPTION
**Pull request by tomerlevi444**
@yuval-netanel @grozen

### CODE REVIEW

File: klarna-on-demand/src/main/java/com/klarna/ondemand/Context.java [hunk 1]
```diff
@@ -24,4 +26,29 @@ protected static String getApiKey(){
         }
         return apiKey;
     }
+
+    /**
+     * Sets the button color for Klarna views in this application.
+     * @param buttonColor Merchant's public API key for this application.
+     */
+    public static void setButtonColor(Integer buttonColor) { Context.buttonColor = buttonColor; }
```
Comment (grozen) [inline klarna-on-demand/src/main/java/com/klarna/ondemand/Context.java:34]: Is it customary for colors to be an integer? How does that work, exactly? Some sort of RGB thing?

File: klarna-on-demand/src/main/java/com/klarna/ondemand/HelperMethods.java [hunk 2]
```diff
@@ -5,4 +5,12 @@
     static Boolean isBlank(String str) {
         return str == null || str.isEmpty();
     }
+
+    static String hexStringFromColor(Integer color) {
```
Comment (grozen) [inline klarna-on-demand/src/main/java/com/klarna/ondemand/HelperMethods.java:9]: I just saw this guy now. This is starting to smell a little non-trivial. Just be sure people end up being able to easily understand how to change the color of various things. Documentation is your friend and all that.

File: klarna-on-demand/src/main/java/com/klarna/ondemand/Context.java (hunk 1 above)
Comment (tomerlevi444) [inline klarna-on-demand/src/main/java/com/klarna/ondemand/Context.java:34]: Apparently in Android development it is very common to use colors as ints.

Comment (MaTriXy) [inline klarna-on-demand/src/main/java/com/klarna/ondemand/Context.java:34]: Colors are treated as resources - integers are used as the ID for that color from the resources mapping. 
http://developer.android.com/reference/android/content/res/Resources.html#getColor(int)

**Pull request merged**